packages/test/test-end-to-end-tests/package.json

```diff
@@ -25,7 +25,7 @@
     "lint:fix": "npm run eslint:fix",
     "start:tinylicious:test": "tinylicious > tinylicious.log 2>&1",
     "test": "npm run test:realsvc",
-    "test:realsvc": "npm run test:realsvc:local && echo 'Skipped: npm run test:realsvc:tinylicious (Issue #6006)'",
+    "test:realsvc": "npm run test:realsvc:local && npm run test:realsvc:tinylicious",
```

Comment:
This is great.  I do wonder how long this is going to add to our PR/CI time.

Reply:
From a cursory glance, it looks like re-enabling would add ~7 minutes. Not sure that I'll get it to the point of re-enabling as part of this PR but we'll see as I go

Reply:
I am ok with that increase for now to get the test coverage.  Please log an item to see if we can parallelize.

Reply:
Created AB#298 to track parallelizing these tests